jug-intro.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><title>JUG Ingolstadt e.V.</title><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui" name="viewport"><link href="node_modules/reveal.js/css/reveal.css" rel="stylesheet"><link rel="stylesheet" href="node_modules/reveal.js/css/theme/white.css" id="theme"><link href="node_modules/reveal.js/lib/css/zenburn.css" rel="stylesheet"><!--[if lt IE 9]><script src="node_modules/reveal.js/lib/js/html5shiv.js"></script><![endif]--><link rel="stylesheet" href="talks/_theme/theme.css"></head><body>
```

Comment:
this file is generated? or why it's in the `.gitignore`?

Reply:
Usually html files are generated from the asciidoc sources and therefore ignored. The intro slide is manually crafted, so it needs to be excluded from `.gitignore`.

Reply:
I see :)
can you format the file as well? 😁

Reply:
_Just_ for you.